scripts/prepare_client.sh

```diff
@@ -70,8 +70,8 @@ cd "${GOPATH}/src/github.com/google/keytransparency"
 go build ./cmd/keytransparency-client
 
 # Generate .keytransparency.yaml file.
-KTYAML="ct-key: \"../certificate-transparency/test/testdata/ct-server-key-public.pem\"
-ct-url: \"http://107.178.246.112\"
```

Comment:
Also, can you please modify `.keytransparency.yaml` to reflect the same default values?